io.github.twisterarmy.Twister.metainfo.xml

```diff
@@ -0,0 +1,27 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop-application">
```

Comment:
why is this desktop-app while the finish-args has no display access?

Reply:
steam (the app) doesn't open in a browser though. it'd be nice if you can get close to an actual app, maybe embedding a webview until a proper client is developed.